lib/honeybadger/util/app_http.rb

```diff
@@ -0,0 +1,44 @@
+require 'honeybadger/util/http'
+require 'base64'
+
+module Honeybadger
+  module Util
+    class AppHTTP < HTTP
+      def personal_auth_headers
+        encoded_token = Base64.encode64("#{config[:personal_auth_token]}:")
+        {"Authorization" => "Basic #{encoded_token}"}
+      end
+
+      def http_headers(headers = nil)
+        {}.tap do |hash|
+          hash.merge!(HEADERS)
```

Comment:
- Content-Type - Needed
- Content-Encoding - Works with the Rest API. Maybe it would be desirable to switch it off as it makes testing a tiny bit easier, on the other hand, you know, compression.
- Accept: Makes sense to me?
- User-Agend: Makes sense, I guess?